test/e2e/atlas/profile_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 MongoDB Inc
```

Comment:
```suggestion
// Copyright 2024 MongoDB Inc
```
🍾

Reply:
Thanks, didn't notice the year when copy-pasting